src/app/shared/table/table.component.html

```diff
@@ -140,6 +140,9 @@
         <div *ngIf="dataSource.isSearchEnabled" class="col search-form">
           <mat-form-field>
             <input appAutofocus autocomplete="off" #searchInput [placeholder]="searchPlaceholder | translate" matInput>
+            <button mat-icon-button matSuffix (click)="resetSearchFilter()" aria-label="Add">
```

Comment:
I think the aria-label does not really match the purpose of the button.

Reply:
True but if you look at line number 113 and line number 121 which both refer to the same button but for different elements, the aria-label used is "Add". I copied it to keep the format.

Reply:
I can change it to "Clear".

Reply:
Yes I think this is also not correct. You can change the aria-labels there too.